backend/Dockerfile

```diff
@@ -5,7 +5,10 @@ WORKDIR /app
 COPY requirements.txt .
 RUN pip install -r requirements.txt
 
-COPY  . /app/backend/.
+RUN apt-get update && apt-get install -y clamav clamav-daemon
```

Comment:
During Docker build, getting this error:


RUN apt-get update && apt-get install -y clamav clamav-daemon

/bin/sh: apt-get: not found 

See comment above, about including clamav modules in the requirements.txt file.